lms/djangoapps/commerce/api/v1/tests/test_views.py

```diff
@@ -18,6 +18,7 @@
 from common.djangoapps.course_modes.models import CourseMode
 from lms.djangoapps.verify_student.models import VerificationDeadline
 from common.djangoapps.student.tests.factories import UserFactory
+from openedx.adg.lms.applications.constants import MAXIMUM_YEAR_OPTION
```

Comment:
Let's please just back port whatever edX has done. Also the core change in the wiki should be updated accordingly.

Reply:
Good catch this is an un-used import. Forgot to remove this after back port.